<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cardaz_app</class>
 <widget class="QMainWindow" name="cardaz_app">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>348</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>cardaz_app</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="form_tabcontrol">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>691</width>
      <height>301</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="login_form">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>100</y>
        <width>271</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Do you have a record with us aleady?</string>
      </property>
     </widget>
     <widget class="QPushButton" name="yesacc_BTN">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>180</y>
        <width>91</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Yes</string>
      </property>
     </widget>
     <widget class="QPushButton" name="noacc_BTN">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>180</y>
        <width>91</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>No</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="P_info_form">
     <widget class="QLineEdit" name="first_name_TB">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>30</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="fname_label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;First Name:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="last_name_TB">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>30</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lname_label">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>30</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Last Name:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="mi_TB">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>30</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="mi_label">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>30</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>mi:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="phone_TB">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>60</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="phone_Label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Phone#: </string>
      </property>
     </widget>
     <widget class="QLabel" name="zipcode_Label">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>150</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Zip Code:</string>
      </property>
     </widget>
     <widget class="QLabel" name="street_Label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Street Address:</string>
      </property>
     </widget>
     <widget class="QLabel" name="state_Label">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>150</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>State:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="phone_TB_2">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>150</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="city_Label">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>150</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>City:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="street_TB">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>120</y>
        <width>531</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="state_CB">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>150</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
      <item>
       <property name="text">
        <string>&lt;None&gt;</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Alabama</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Alaska</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Arkansas</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>California</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Colorado</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Connecticut</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Delaware</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Florida</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Georgia</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Hawaii</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Idaho</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Illinois</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Indiana</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Iowa</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Kansas</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Kentucky</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>louisiana</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Maine</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Maryland</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Massachusetts</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Michigan</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Minnesota</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mississippi</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Missouri</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Montana</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Nebraska</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Nevada</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>New Hamshire</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>New Jersey</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>New Mexico</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>New York</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>North Carolina</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>North Dakota</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Ohio</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Oklahoma</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Oregon</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Pennsylvannia</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Rhode Island</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>South Carolina</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Tennessee</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Texas</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Utah</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Vermont</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Virginia</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>West Virginia</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Wisconsin</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Wyoming</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="street_TB_2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>150</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="titlelLabel">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>0</y>
        <width>181</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Patient infomation Form</string>
      </property>
     </widget>
     <widget class="QPushButton" name="execute_BTN">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>220</y>
        <width>91</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="select_account">
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>641</width>
        <height>201</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>test 1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>test 2</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="start_info_Label">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>20</y>
        <width>411</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Please Select your account, and hit ready to start the test.</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ready_BTN">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>250</y>
        <width>91</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Ready</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="hgraph">
     <widget class="QPushButton" name="back_BTN">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>0</y>
        <width>91</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>back</string>
      </property>
     </widget>
     <widget class="PlotWidget" name="heart_graph">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>671</width>
        <height>271</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header location="global">pyqtgraph.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
